templates/test/overview.html.ep

```diff
@@ -11,10 +11,10 @@
     <div id="summary" class="panel <%= ($aggregated->{failed} + $aggregated->{incomplete}) ? 'panel-danger' : 'panel-success' %>">
         <div class="panel-heading">
             Overall Summary of
-            % if ($group) {
-                <b><%= link_to $group->name => url_for('group_overview', groupid => $group->id) %></b>
-            % } else
-            % {
+            % if (@$groups) {
+                <b><%= b join(', ', map { link_to $_->name => url_for('group_overview', groupid => $_->id) } @$groups) %></b>
```

Comment:
Please see the following screenshot showing a combined query for multiple groups (which are even covering multiple distri's as well as selecting multiple flavors):
![openqa_multiple_groupids_in_overview](https://cloud.githubusercontent.com/assets/1693432/21434727/59e90702-c876-11e6-8b17-67c399a2c76a.png)